### PR TITLE
dropped contrib/simplejson

### PR DESCRIPTION
**Pull request by ilvalle**

### CODE REVIEW

File: pydal/_load.py [hunk 1]
```diff
@@ -5,10 +5,12 @@
     from collections import OrderedDict
 except:
     from .contrib.ordereddict import OrderedDict
+    
 try:
+# try external module
+    import simplejson as json
```
Comment (gi0baro) [inline pydal/_load.py]: why keep trying to load simplejson if we're dropping py25?

Comment (ilvalle) [inline pydal/_load.py]: as far as I remember, the external module (if present) is faster

 Paolo

2015-04-27 0:04 GMT+02:00 Giovanni Barillari <notifications@github.com>:

Comment (gi0baro) [inline pydal/_load.py]: From simplejson page: 
" simplejson is the externally maintained development version of the json library included with Python 2.6 and Python 3.0, but maintains backwards compatibility with Python 2.5."

Btw, this is already patched in py3 branch.

Comment (niphlod) [inline pydal/_load.py]: simplejson is updated more frequently and is generally faster than the json module shipped with python. It's not going to hurt anybody if you leave that line, importing simplejson if it's found and fallbacking to json.

**Comment by gi0baro** [comment]
Since this is already done in py3 and I'm gonna merge it, I'm closing this

**Pull request closed without merging**